### PR TITLE
Risk of Rain, ADE Pulse Turrets patch

### PR DESCRIPTION
**Pull request by Theblessedwant**
## Additions

Patch Compatiblity for Risk of Rain UE Light Contact Armory and ADE Pulse Turrets

## Changes

Fixed Rimmunation Rangers Pathing 

## References

nil

## Alternatives

Describe alternative implementations you have considered, e.g.
Removed modded ammo fthat came with ADE Pulse turrets to reduce vonusion

## Testing

Check tests you have performed:
- [y] Compiles without warnings
- [y] Game runs without errors
- [-] (For compatibility patches) ...with and without patched mod loaded
- [-] Playtested a colony (specify how long)

### CODE REVIEW

**Comment by Theblessedwant** [comment]
Please Balance these, I did not really take into consideration the Balance for the ADE pulse turrets. The armors in the Rangers Patch and the UE Light Armory patch also dont have different armor levels yet

File: Patches/RH2 Faction - The Rangers/Rangers_CE_Patch_Apparel.xml [hunk 1]
```diff
@@ -84,7 +82,7 @@
             <equippedStatOffsets>
                 <PsychicSensitivity>-0.2</PsychicSensitivity>
                 <AimingAccuracy>0.15</AimingAccuracy>
-                <ToxicEnvironmentResistance>0.50</ToxicEnvironmentResistance>
```
Comment (SamaelGray) [inline Patches/RH2 Faction - The Rangers/Rangers_CE_Patch_Apparel.xml:87]: These should stay as `ToxicEnvironmentResistance`, `ToxicResistance` is used elsewhere and protects against venom bites and other similar damage sources.

**Review by SamaelGray — CHANGES_REQUESTED** [review]

**Comment by Theblessedwant** [comment]
Error with the Specialist Armor from Risk of Rain in the UI, trying to diagnose the issue

File: Patches/RH2 Faction - The Rangers/Rangers_CE_Patch_Apparel.xml (hunk 1 above)
Comment (Theblessedwant) [reply Patches/RH2 Faction - The Rangers/Rangers_CE_Patch_Apparel.xml]: Changed all to Toxic EnvironmentResistance

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
The Risk of Rain patches appear fine.

However, why do the various pulse turrets not have magazines or consume ammo?

**Comment by Theblessedwant** [comment]
> The Risk of Rain patches appear fine.
> 
> However, why do the various pulse turrets not have magazines or consume ammo?

Fixed, forgot to add Ammouser

**Review by SamaelGray — APPROVED** [review]

**Pull request merged**